api.js

```diff
@@ -159,7 +164,6 @@ RosterAPI.prototype.request = function(path, opts) {
     var fullURL = utils.joinURL(this.url, path);
 
     this.requestLog.push([fullURL, opts]);
-    this.saveState();
```

Comment:
@gregstuderoaf  other small update -- I think this is an unnecessary call to `saveState` -- nothing in this function is updating a field in `this` (except for `dataTableAttach` -- which calls `saveState` on its own).

Reply:
Yeah, this call was only here since I used to save the requestLog - that ends up causing an issue though because Telerivet has a very small limit on the amount of string/int data you can save in the state variables before erroring out.